lib/orcid/add_works.rb

```diff
@@ -42,8 +42,8 @@ def add_work(author, contribution, orcid_user)
       contribution.save!
       logger&.info("#{self.class} - author #{author.id} - added publication #{contribution.publication.id} with put-code #{contribution.orcid_put_code}")
       true
-    rescue Orcid::PubMapper::PubMapperError => e
-      logger&.info("#{self.class} - author #{author.id} - did not add publication #{contribution.publication.id}: #{e.message}")
+    rescue Orcid::PubMapper::PubMapperError, Orcid::Client::InvalidTokenError => e
+      logger&.warn("#{self.class} - author #{author.id} - did not add publication #{contribution.publication.id}: #{e.message}")
```

Comment:
1. catch this new exception as one that will not get an HB notification (we will throw the new exception below when necessary)
2. also, seems to make more sense to log this as a warn instead of an info to make it easier to grep this out later?